contracts/interfaces/CTokenInterfaces.sol

```diff
@@ -0,0 +1,445 @@
+pragma solidity >=0.5.16;
```

Comment:
I don't recommend putting every interface in the same directory as this is an external interface we should make another directory called `external` which should have the external interfaces or contracts. This will be much easier for auditors to find which contracts they need to audit and it will be easy for us to manage as well. 

Although instead of copy-pasting we could use the npm package to avoid human error and it will automatically break the compilation if the interface gets changed from the upstream. 

Another way is to have a custom interface where we only keep those functions that we are using to keep it clean and understandable.